pyuvdata/parameter.py

```diff
@@ -974,6 +991,7 @@ def set_lat_lon_alt_degrees(self, lat_lon_alt_degree):
                 longitude * np.pi / 180.0,
                 altitude,
                 frame=self.frame,
+                lunar_ellipsoid=self.lunar_ellipsoid,
             )
 
     def check_acceptability(self):
```

Comment:
Does `check_acceptability` here need to be updated for what values of `frame` are okay?

Reply:
hmm, maybe. I think there's checking on the setter, but I'll make sure.

Reply:
Oh, right. The setter was removed when we moved the checking for positions being on the surface to the utils. I'm a bit inclined to add a setter to check that it's allowed, because it's used in so many calculations downstream.